src/Pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import "./HomePage.css"
+
+import Navbar from "../../Components/Navbar/Navbar"
+import Footer from "../../Components/Footer/Footer"
+
+import DealsSection from './PageSections/DealsSection';
+import BrandSection from './PageSections/BrandsSection.js'
+import ArrivalsSection from './PageSections/ArrivalsSection.js'
+import CategoriesSection from './PageSections/CategoriesSection';
+import HeroSection from './PageSections/HeroSection'
```

Comment:
suggestion: add an index.js file in PagesSections and export all sections from index.js

Reply:
done